src/transform.ts

```diff
@@ -54,27 +51,27 @@ export function getTransactionFromBlockPayout(block: algosdk.modelsv2.BlockRespo
  * @param blockResponse An Algorand block
  * @returns An array of processed transactions from the block
  */
-export function getBlockTransactions(blockResponse: algosdk.modelsv2.BlockResponse): TransactionInBlock[] {
+export function getBlockTransactions(blockResponse: BlockResponse): TransactionInBlock[] {
   const block = blockResponse.block
 
   let roundOffset = 0
   const getRoundOffset = () => roundOffset++
 
   const txns = (block.payset ?? []).flatMap((signedTransactionInBlock) => {
     // Assume that Consensus.RequireGenesisHash is true, we always copy the genesis hash from the block to the transaction
-    const genesisHash = Buffer.from(blockResponse.block.header.genesisHash)
-    const genesisId = signedTransactionInBlock.hasGenesisID ? blockResponse.block.header.genesisID : undefined
+    const genesisHash = block.header.genesisHash ? Buffer.from(block.header.genesisHash) : undefined
```

Comment:
Latest utils-ts should account for patching the transaction with these value, so we should be able to do away with this code. Double check this is working correctly though.